rbac/management/group/view.py

```diff
@@ -147,13 +147,18 @@ def get_serializer_class(self):
         return GroupSerializer
 
     def protect_default_groups(self, action):
-        """Deny modifications on platform_default groups."""
+        """Deny modifications on platform_default and admin_default groups."""
         group = self.get_object()
         if group.platform_default:
             key = "group"
             message = "{} cannot be performed on platform default groups.".format(action.upper())
             error = {key: [_(message)]}
             raise serializers.ValidationError(error)
+        if group.admin_default:
```

Comment:
Sort of a nit, but I wonder if we could reuse the check above on 152, and just have it look for platform _or_ admin default, and dynamically set the failure message (or generalize it), to keep it one condition?

Reply:
Ok I generalized the message to say the action 'cannot be performed on default groups'.